modin/experimental/core/execution/native/implementations/omnisci_on_native/test/test_dataframe.py

```diff
@@ -1723,6 +1724,7 @@ class TestDateTime:
         "c": pandas.to_datetime(
             ["20190902", "20180913", "20190921", "20180903"], format="%Y%m%d"
         ),
+        "d": pandas.to_datetime(["2018-10-26 12:00", "2018-10-26 13:00:15"]),
```

Comment:
```suggestion
        "d": pandas.to_datetime(
            [
                "2018-10-26 12:00",
                "2018-10-26 13:00:15",
                "2020-10-26 04:00:15",
                "2020-10-26",
            ]
        ),
```
the "d" column had only 2 elements when others have 4. This made the test fail silently at the DataFrames construction stage: [this piece of code](https://github.com/modin-project/modin/blob/0f5a47177025b069e74de9d8caff17e1e6b24fec/modin/experimental/core/execution/native/implementations/omnisci_on_native/test/utils.py#L261-L278) tries to build modin and pandas dataframes from the given data and considers that the test passes if they both raise an error (always happen if the data is incorrect).

I think we have to add a parameter or smth else in a separate PR to explicitly turn this behavior on, so no implicit silent errors occur.

Reply:
Yeah, my bad...